amd/hipcc/utils.cmake

```diff
@@ -0,0 +1,125 @@
+## Configure Copyright File for Debian Package
+function( configure_pkg PACKAGE_NAME_T COMPONENT_NAME_T PACKAGE_VERSION_T MAINTAINER_NM_T MAINTAINER_EMAIL_T)
+    # Check If Debian Platform
+    find_file (DEBIAN debian_version debconf.conf PATHS /etc)
+    if(DEBIAN)
+      set( BUILD_DEBIAN_PKGING_FLAG ON CACHE BOOL "Internal Status Flag to indicate Debian Packaging Build" FORCE )
+      set_debian_pkg_cmake_flags( ${PACKAGE_NAME_T} ${PACKAGE_VERSION_T}
+                                  ${MAINTAINER_NM_T} ${MAINTAINER_EMAIL_T} )
+
+      # Create debian directory in build tree
+      file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/DEBIAN")
+
+      # Configure the copyright file
+      configure_file(
+        "${CMAKE_SOURCE_DIR}/DEBIAN/copyright.in"
+        "${CMAKE_BINARY_DIR}/DEBIAN/copyright"
+        @ONLY
+      )
+
+      # Install copyright file
+      install ( FILES "${CMAKE_BINARY_DIR}/DEBIAN/copyright"
+	        DESTINATION "${CMAKE_INSTALL_DOCDIR}"
+	        COMPONENT ${COMPONENT_NAME_T} )
+
+      # Configure the changelog file
+      configure_file(
+        "${CMAKE_SOURCE_DIR}/DEBIAN/changelog.in"
+        "${CMAKE_BINARY_DIR}/DEBIAN/changelog.Debian"
+        @ONLY
+      )
+
+      if( BUILD_ENABLE_LINTIAN_OVERRIDES )
+	if(NOT ${BUILD_SHARED_LIBS})
```

Comment:
I don't think we need this section.  We do not have a build version of HIPCC with shared libraries, nor asan.